committee-steering/sig-governance.md

```diff
@@ -0,0 +1,533 @@
+through contributions in PRs, issue triage, test triage, operations,
+etc.
+
+Members are expected to be responsive to issues for sub areas
+they own within the SIG.  Members are expected to help other
+members in the SIG (e.g. transfer knowledge, help debug, assist with
+operational tasks, answer questions, etc).
+
+Membership is provisional until the member has shown sustained
+contributions for 3 months.  Provisional members may provide
+feedback in decision making, but are not deciders in the decision
+itself.
+
+## Processes and tools
+
+The following section outlines various processes used by the SIG.
+
+### Decision making within the SIG
+
+It is critical to a SIG that the decision making processes is healthy.
+A good decision making process has the following properties:
+
+- Decision making process is unambiguously understood by everyone involved
+- Decisions are resolved instead of languishing and defaulting to no decision
+- Decisions are a commitment and won't waffle after having been made
+
+#### Who, What, Where, When, How
+
+In cases where a proposal author is seeking acceptance from the
+community it is important to have a clearly defined process for
+communication and resolution of the proposal.
+
+While it is not necessary for all decisions to go through the lazy
+consensus through the full SIG, it is important that such a process
+exists for decisions that may affect many members of the SIG.
+
+When using [lazy consensus][lazy-consensus], all proposals must be published to a visible
+location.  For acceptance to be given be implicitly (as it is in
+lazy consensus models), all involved parties must be made aware of the
+proposal.  While it is the responsibility of involved parties to
+watch for new proposals and read them in a timely manner,
+it is the responsibility of the SIG governance to ensure that
+where and how are well defined.
+
+In order to avoid [Warnock's Dilemma][warnocks-dilemma], it is
+recommended at least one active member respond to a proposal before
+it is accepted.
+
+In SIG ~~Foo~~, proposals are made using a commonly used tool
+(issues, PRs, docs, etc) and then published to the SIG
+through kubernetes-sig~~foo~~-members@googlegroups.com mailing list with the
+subject containing either `KEP: ` or `RFC: `.  This provides a simple
+mechanism to subscribe to proposals that is less noisy than subscribing
+to github notifications.
+
+Discussion may occur on the issue / PR / doc itself.  Any comments
+on the doc are considered blocking until they are responded too.
+Once the comments are responded to, they are considered addressed
+unless either there is another response *or* the comment has some
+language to indicate it is a blocking issue that must be explicitly
+resolved.  e.g. *lets hold off on doing this until...*
+
+Once all comments have been addressed on the proposal without responses,
+another email
+must be sent to the mailing list with `LastCall: KEP: ` or
```

Comment:
We're baking in too much process and unless this is automated, it will be circumvented or ignored.

Reply:
How do you think the process should work?

Reply:
I made this optional, saying that there is a high commitment to not require addressing later feedback if this is sent